dlc_developer_config.toml

```diff
@@ -59,13 +59,18 @@ use_scheduler = false
 # dlc-pr-huggingface-pytorch = "huggingface/pytorch/buildspec-1-9.yml"
 # Setting the buildspec file path to "" allows the image builder to choose the default buildspec file.
 
-# Standard Framework DLC buildspecs
-dlc-pr-autogluon = ""
+# E3 Framework DLC buildspecs
 dlc-pr-mxnet = ""
```

Comment:
Should we rename this to `dlc-pr-ec2-pytorch`?

Reply:
No, the pre-existing CI project names have not been changed. So we currently have `dlc-pr-pytorch` and `dlc-pr-sagemaker-pytorch`.

Reply:
Do you believe that the older projects should be renamed?

Reply:
yes, lets change the project names to be explicitly clear